Makefile

```diff
@@ -6,35 +6,42 @@ NAMESPACE ?= nmstate-default
 IMAGE_REGISTRY ?= quay.io/nmstate
 IMAGE_TAG ?= latest
 PULL_POLICY ?= Always
-STATE_HANDLER_IMAGE ?= kubernetes-nmstate-state-handler
-POLICY_HANDLER_IMAGE ?= kubernetes-nmstate-configuration-policy-handler
+OPERATOR_IMAGE ?= kubernetes-nmstate
+NMSTATECTL_IMAGE ?= nmstatectl
 KUBECONFIG ?= $(shell pwd)/cluster/.kubeconfig
 
-commands = state-handler policy-handler
-func_tests_sources = $(shell find tests/ -name "*.go")
-apis_sources = $(shell find pkg/apis -name "*.go")
+manager_bin = build/_output/bin/kubernetes-nmstate
+func_tests_sources = $(wildcard tests/*.go)
+apis_sources = $(wildcard pkg/apis/**/*.go)
+pkg_sources = $(wildcard pkg/**/*.go)
+cmd_sources = $(wildcard find cmd/**/*.go)
 
-$(commands):
-	go fmt ./cmd/$@
-	go vet ./cmd/$@
-	go build -o ./bin/$@ ./cmd/$@ 
-
-build: $(commands)
+$(manager_bin): $(pkg_sources) $(cmd_sources)
+	go fmt ./cmd/manager
+	go vet ./cmd/manager
+	CGO_ENABLED=0 GOOS=linux go build \
+		-o $@ ./cmd/manager/
 
 tests/tests.test: $(func_tests_sources) $(apis_sources)
 	ginkgo build tests
 
-docker-state-handler:
-	docker build -f cmd/state-handler/Dockerfile -t $(IMAGE_REGISTRY)/$(STATE_HANDLER_IMAGE):$(IMAGE_TAG) .
+operator: $(operator_bin)
```

Comment:
I would try to avoid calling it operator.

> An Operator is a method of packaging, deploying and managing a Kubernetes application. A Kubernetes application is an application that is both deployed on Kubernetes and managed using the Kubernetes APIs and kubectl tooling.

https://coreos.com/operators/

We however, don't use the controller to deploy the application, our controller is **the** application.

So we need two names, one for the central controller and one for nmstatectl DS. We should not call the central part controller, since it will consist of multiple controllers under a single manager (ouch). Do you have any names in mind? Maybe we can use "manager" for the central part. Then something opposing to manager for the DS. "worker" maybe? Although I'm not sure it would be considered a worker, maybe "agent"?